### PR TITLE
POC: Holdings record simple entity type

### PR DESCRIPTION
**Pull request by ncovercash**

### CODE REVIEW

File: src/main/resources/entity-types/inventory/simple_holdings_records.json5 [hunk 1]
```diff
@@ -0,0 +1,740 @@
...
+            filterValueGetter: "( SELECT array_agg(lower(elems.value->>'displayType')) FROM jsonb_array_elements(:sourceAlias.jsonb->'receivingHistory') AS elems)",
+            valueFunction: 'lower(:value)',
+          },
+          {
+            name: 'entries',
+            dataType: {
+              dataType: 'arrayType',
+              itemDataType: {
+                dataType: 'objectType',
+                properties: [
```
Comment (ncovercash) [inline src/main/resources/entity-types/inventory/simple_holdings_records.json5:591]: we have our first doubly-nested object! 🤮

Comment (kjain110) [reply src/main/resources/entity-types/inventory/simple_holdings_records.json5:591]: tbh I am quite interest in seeing the ref doc structure for it. :)

Comment (ncovercash) [reply src/main/resources/entity-types/inventory/simple_holdings_records.json5:591]: https://github.com/folio-org/mod-inventory/blob/53abb78ebf6f54e8fbba85cdff7f6c18e54e3ea7/ramls/holdings-record.json#L210-L240

**Review by kjain110 — APPROVED** [review]

**Review by bvsharp — APPROVED** [review]

File: src/main/resources/entity-types/inventory/simple_holdings_records.json5 [hunk 2]
```diff
@@ -0,0 +1,740 @@
...
+      dataType: {
+        dataType: 'arrayType',
+        itemDataType: {
+          dataType: 'rangedUUIDType',
+        },
+      },
+      isIdColumn: false,
+      queryable: false,
+      visibleByDefault: false,
+      valueGetter: "(\n  SELECT\n    array_agg(el::text)\n  FROM\n    jsonb_array_elements_text(:sourceAlias.jsonb->'statisticalCodeIds'::text) el (value)\n)",
```
Comment (mweaver-ebsco) [inline src/main/resources/entity-types/inventory/simple_holdings_records.json5:259]: Nit: We don't need these `\n`s

File: src/main/resources/entity-types/inventory/simple_holdings_records.json5 (hunk 1 above)
Comment (ncovercash) [reply src/main/resources/entity-types/inventory/simple_holdings_records.json5]: oops, the outer one wasn't an array-object, just an object which should've been flattened. fixed.

**Pull request merged**